db/queries.js

```diff
@@ -0,0 +1,20 @@
+const {db, Questions, Answers, Photos } = require('../sequelize/connection.js');
+
+
+// Answers practice query
+const getAnswers = async (questionId) => {
+  let answers = await Answers.findAll({
+    where: {questions_id: questionId}
+  })
+  console.log(answers.dataValues)
+  for (var answer of answers) {
+    console.log(answer.dataValues)
+  }
+  return answers;
+}
+
+// console.log('answers', getAnswers());
+
+getAnswers(4)
```

Comment:
Hey if you were to console log what is return from this function invocation, what would it be? I've been meaning to test out how this situation works with async/await but haven't gotten around to it.